projects/plugins/jetpack/modules/search/instant-search/components/path-breadcrumbs.jsx

```diff
@@ -8,10 +8,16 @@ import './path-breadcrumbs.scss';
 function splitDomainPath( path ) {
 	const splits = path.split( '/' ).filter( piece => piece.length > 0 );
 	splits.shift(); // Removes domain name from splits; e.g. 'jetpack.com'
-	return splits.length === 0 ? [ '/' ] : splits;
+	return splits.length === 0 ? [] : splits;
 }
 
 const PathBreadcrumbs = ( { className, onClick, url } ) => {
+	const breadcrumbPieces = splitDomainPath( url );
```

Comment:
L28 could share variable `breadcrumbPieces` to avoid another `splitDomainPath` call.

Reply:
Absolutely! [Updated here](https://github.com/Automattic/jetpack/pull/20500/commits/e2fd6c2e454ed4093d4de946e7394d5ac7b77558)